statsmodels/examples/ex_ordered_model.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 24 11:17:06 2015
```

Comment:
Could you do a notebook? We don't really use ex_... Python files anymore and they dont' show up in the docs.